mysql-test/include/backup_locks_binlog.inc

```diff
@@ -258,8 +258,14 @@ INSERT INTO tt_csv VALUES(5);
 INSERT INTO tt_blackhole VALUES(5);
 INSERT INTO tt_archive VALUES(5);
 
+--connection default
+--echo # connection default
+
 UNLOCK TABLES;
 
+--connection con1
+--echo # connection con1
+
 COMMIT;
```

Comment:
Is this COMMIT still required in con1? There does not seem to be an open transaction there